examples/platform/silabs/SiWx917/SiWx917/rs911x.gni

```diff
@@ -15,60 +15,58 @@ rs911x_src_plat = [
+  "${wisemcu_sdk_root}/platforms/si91x/hal/src/rsi_hal_mcu_m4_rom.c",
+  "${wisemcu_sdk_root}/platforms/si91x/hal/src/rsi_hal_mcu_interrupt.c",
 ]
 rs911x_plat_incs = [
   "${wifi_sdk_dir}",
   "${wifi_sdk_dir}/hal",
-  "${wiseconnect_sdk_root}/platforms/si91x/hal/inc",
+  "${wisemcu_sdk_root}/platforms/si91x/hal/inc",
 
-  #  "${wiseconnect_sdk_root}/sapi/include",
   "${chip_root}/src/platform/silabs/SiWx917",
   "${chip_root}/src/platform/silabs/SiWx917/bluetooth",
   "${examples_plat_dir}/SiWx917",
-  "${wiseconnect_sdk_root}/platforms/si91x/hal/inc",
+  "${wisemcu_sdk_root}/platforms/si91x/hal/inc",
 ]
 
 #
 # All the stuff from wiseconnect
 #
 rs911x_src_sapi = [
-  "${wiseconnect_sdk_root}/sapi/wlan/rsi_wlan_apis.c",
-  "${wiseconnect_sdk_root}/sapi/common/rsi_apis_non_rom.c",
-  "${wiseconnect_sdk_root}/sapi/common/rsi_apis_rom.c",
-  "${wiseconnect_sdk_root}/sapi/common/rsi_common_apis.c",
-  "${wiseconnect_sdk_root}/sapi/common/rsi_device_init_apis.c",
+  "${wisemcu_sdk_root}/sapi/wlan/rsi_wlan_apis.c",
+  "${wisemcu_sdk_root}/sapi/common/rsi_apis_non_rom.c",
+  "${wisemcu_sdk_root}/sapi/common/rsi_apis_rom.c",
+  "${wisemcu_sdk_root}/sapi/common/rsi_common_apis.c",
+  "${wisemcu_sdk_root}/sapi/common/rsi_device_init_apis.c",
 
-  #  "${wiseconnect_sdk_root}/sapi/common/rsi_wisemcu_hardware_setup.c",
-  "${wiseconnect_sdk_root}/sapi/driver/device_interface/spi/rsi_spi_frame_rd_wr.c",
-  "${wiseconnect_sdk_root}/sapi/driver/device_interface/spi/rsi_spi_functs.c",
-  "${wiseconnect_sdk_root}/sapi/driver/device_interface/spi/rsi_spi_iface_init.c",
-  "${wiseconnect_sdk_root}/sapi/driver/device_interface/spi/rsi_spi_mem_rd_wr.c",
-  "${wiseconnect_sdk_root}/sapi/driver/device_interface/spi/rsi_spi_reg_rd_wr.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_common.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_device_init.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_driver_event_handlers.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_driver.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_events_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_events.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_iap.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_nwk_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_nwk.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_pkt_mgmt_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_pkt_mgmt.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_queue_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_queue.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_scheduler_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_scheduler.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_setregion_countryinfo.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_timer.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_utils_rom.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_utils.c",
-  "${wiseconnect_sdk_root}/sapi/driver/rsi_wlan.c",
-  "${wiseconnect_sdk_root}/sapi/rtos/freertos_wrapper/rsi_os_wrapper.c",
```

Comment:
You guys are fragmenting your SDK codebase , i have seen so many PRs for new module this is becoming very hard to follow. all of this doesn't give confidence on having a stable platform in order to build thing

Reply:
@jmeg-sfy Just i have renamed SDK root path form wiseconnect_sdk_root to wisemcu_sdk_root.

Reply:
@jmeg-sfy I understand it is getting complicated a lot to review the multiple PR's in which the modification of the build system is there. There will be a follow up PR for the cleanup of the build files, which will be limit the modifications on the build system. And that should be the last for the major modification.